demo/main.scss

```diff
@@ -0,0 +1,2 @@
+@import 'styles/month-view';
+@import 'styles/week-view';
```

Comment:
Can you move all scss files into the root directory, call it scss and rename main to `angular2-calendar` like so:
```
    ├── demo
    └── scss
        ├── angular2-calendar.scss
        ├── month-view.scss
        ├── week-view.scss
        ├── variables.scss
``` 

As demo isn't published to npm so people consuming the module wouldn't be able to import the scss files. This way they can import like:
```
@import 'angular2-calendar/scss/angular2-calendar';
```